src/toast/Toast.tsx

```diff
@@ -0,0 +1,55 @@
+import "./_toast.scss";
+
+import React, {useEffect} from "react";
+import ReactDOM from "react-dom";
+import classNames from "classnames";
+
+import {useToast} from "./util/toastHooks";
+
+function Toast() {
+  const {
+    toastState: {
+      isDisplayed,
+      data: {autoClose, timeout, content, mode, customClassName}
+    },
+    dispatchToastAction
+  } = useToast();
+  let toastRootNode = document.querySelector("#toast-root");
+
+  if (!toastRootNode) {
+    toastRootNode = document.createElement("div");
+    toastRootNode.setAttribute("id", "toast-root");
+  }
+
+  document.body.appendChild(toastRootNode);
+
+  useEffect(() => {
+    let timeoutId: number;
```

Comment:
I think `NodeJS.Timeout` is the correct type for this variable

(see return type in the ss)
<img width="1096" alt="Screen Shot 2021-04-14 at 12 48 01 PM" src="https://user-images.githubusercontent.com/22727931/114690800-b881e680-9d1f-11eb-8299-af8e34cc9806.png">